admin/modules/home/index.php

```diff
@@ -260,13 +260,16 @@
 	$query = $db->simple_select("posts", "COUNT(*) AS newposts", "dateline > '$timecut' AND visible='1'");
 	$newposts = my_number_format($db->fetch_field($query, "newposts"));
 
-	// Get the number of reported post
-	$query = $db->simple_select("reportedcontent", "COUNT(*) AS reported_posts", "type = 'post' OR type = ''");
-	$reported_posts = my_number_format($db->fetch_field($query, "reported_posts"));
+	// If report medium is MCP...
```

Comment:
No `if` here either. Besides, only the unread number is redundant due to no "mark as read" option. The other one may still be useful to see how many reports were sent.